kano_init_flow/stages/overscan_old/main.py

```diff
@@ -222,7 +222,13 @@ def disable(self):
 
     def save_changes(self):
         if self._original != self._current:
-            write_overscan_values(self._current)
+            #write_overscan_values(self._current)
```

Comment:
Oh, right. I forgot to remove it. I'll do that tomorrow.

On Wednesday, 28 October 2015, Alex Simon <notifications@github.com> wrote:

> In kano_init_flow/stages/overscan_old/main.py
> <https://github.com/KanoComputing/kano-init-flow/pull/86#discussion_r43296652>
> :
>
> > @@ -222,7 +222,13 @@ def disable(self):
> >
> >      def save_changes(self):
> >          if self._original != self._current:
> > -            write_overscan_values(self._current)
> > +            #write_overscan_values(self._current)
>
> Is this comment necessary?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/KanoComputing/kano-init-flow/pull/86/files#r43296652>.
>


-- 
Radek Pazdera
Software Engineer at Kano <http://www.kano.me>